sale_order_general_discount/readme/CONTRIBUTORS.rst

```diff
@@ -1,3 +1,4 @@
 * Sergio Teruel <sergio.teruel@tecnativa.com>
 * Raf Ven <raf.ven@dynapps.be>
 * Sudhir Arya <sudhir@erpharbor.com>
+* Stefan Ungureanu <stefan.ungureanu@tecnativa.com>
```

Comment:
Can you change our attributions in our current format? Example:

https://github.com/OCA/rma/blame/14.0/rma/readme/CONTRIBUTORS.rst#L1-L5

Reply:
Let me know if I've done it right